docs-src/luna-streaming-core/antora-luna-streaming.yml

```
@@ -1,6 +1,6 @@
 name: docs
 title: DataStax Luna Streaming
-version: 2.8
+version: 2.10
```

This is my fault. Make sure yo put version numbers in single quotes. YAML files don't process trailing `0`'s like this unless they're in single quotes.

```suggestion
version: '2.10'
```